packages/command-palette/src/commands.tsx

```diff
@@ -73,1085 +74,1186 @@ export interface Command {
+				siteSelector: true,
+				siteSelectorLabel: __( 'Select site to visit the homepage', __i18n_text_domain__ ),
+				icon: seenIcon,
+			},
+			openSiteDashboard: {
+				name: 'openSiteDashboard',
+				label: __( 'Open site dashboard', __i18n_text_domain__ ),
+				callback: commandNavigation( '/home/:site' ),
+				searchLabel: [
+					_x(
+						'open site dashboard',
+						'Keyword for the Open site dashboard command',
+						__i18n_text_domain__
+					),
+					_x( 'admin', 'Keyword for the Open site dashboard command', __i18n_text_domain__ ),
+					_x( 'wp-admin', 'Keyword for the Open site dashboard command', __i18n_text_domain__ ),
+				].join( ' ' ),
+				context: [ '/sites' ],
+				siteSelector: true,
+				siteSelectorLabel: __( 'Select site to open dashboard', __i18n_text_domain__ ),
+				icon: dashboardIcon,
+			},
+			openHostingConfiguration: {
+				name: 'openHostingConfiguration',
+				label: __( 'Open hosting configuration', __i18n_text_domain__ ),
+				callback: commandNavigation( '/hosting-config/:site' ),
+				searchLabel: [
+					_x(
+						'open hosting configuration',
+						'Keyword for the Open hosting configuration command',
+						__i18n_text_domain__
+					),
+					_x(
+						'admin interface style',
+						'Keyword for the Open hosting configuration command',
+						__i18n_text_domain__
+					),
+					_x( 'cache', 'Keyword for the Open hosting configuration command', __i18n_text_domain__ ),
+					_x(
+						'database',
+						'Keyword for the Open hosting configuration command',
+						__i18n_text_domain__
+					),
+					_x(
+						'global edge cache',
+						'Keyword for the Open hosting configuration command',
+						__i18n_text_domain__
+					),
+					_x(
+						'hosting',
+						'Keyword for the Open hosting configuration command',
+						__i18n_text_domain__
+					),
+					_x( 'mysql', 'Keyword for the Open hosting configuration command', __i18n_text_domain__ ),
+					_x(
+						'phpmyadmin',
+						'Keyword for the Open hosting configuration command',
+						__i18n_text_domain__
+					),
+					_x(
+						'php version',
+						'Keyword for the Open hosting configuration command',
+						__i18n_text_domain__
+					),
+					_x(
+						'sftp/ssh credentials',
+						'Keyword for the Open hosting configuration command',
+						__i18n_text_domain__
+					),
+					_x(
+						'wp-cli',
+						'Keyword for the Open hosting configuration command',
+						__i18n_text_domain__
+					),
+				].join( ' ' ),
+				context: [ '/sites' ],
+				siteSelector: true,
+				siteSelectorLabel: __( 'Select site to open hosting configuration', __i18n_text_domain__ ),
+				capability: SiteCapabilities.MANAGE_OPTIONS,
+				filterP2: true,
+				filterSelfHosted: true,
+				filterNotice: __( 'Only listing sites hosted on WordPress.com.', __i18n_text_domain__ ),
+				icon: settingsIcon,
+			},
+			openPHPmyAdmin: {
+				name: 'openPHPmyAdmin',
+				get label() {
+					return __( 'Open database in phpMyAdmin', __i18n_text_domain__ );
+				},
```

Comment:
Is this getter actually needed? I don't see we use this pattern for other commands.

Reply:
Thanks for catching it! I was testing something and it seems like I've committed it by mistake.